tests/impulse-tests/Makefile

```diff
@@ -122,10 +122,7 @@ travis:
 #########################################################################
 # automatic github action test
 github_action:
-	$(MAKE) -f Make.rust outdir=rust/osec CARGOOPTIONS="" FAUSTOPTIONS="-I ../../libraries/ -double -os -ec -rnt -a archs/rust/architecture_osecrnt.rs"
```

Comment:
Why do you remove the tests here ? (I think it make sense to keep all of them)

Reply:
The rational was to only keep tests that relate to a change in ci. I don't have a strong opinion on this.